assets/src/scss/5-pages/_api.scss

```diff
@@ -186,6 +186,12 @@ $api-box-padding-y: ceil($font-size-base / 2);
         tbody > tr > th {
             padding-left: $api-box-padding-x;
         }
+
+        tr {
+          a {
+            word-break: break-word;
```

Comment:
The problem was the opposite as far as I understand: words were broken in the middle. And here we're adding more breaks.

Reply:
Thank you, I probably didn't understand, thanks